common/src/configuration/network.rs

```diff
@@ -25,12 +25,18 @@ use std::{
     fmt,
     fmt::{Display, Formatter},
     str::FromStr,
+    sync::Mutex,
 };
 
 use serde::{Deserialize, Serialize};
+use structopt::lazy_static::lazy_static;
 
 use crate::ConfigurationError;
 
+lazy_static! {
+    pub static ref CURRENT_NETWORK: Mutex<Network> = Mutex::new(Network::default());
+}
```

Comment:
Can you use `OnceCell` here instead? Other uses of this pattern elsewhere are moving to it.

Reply:
See #5992, which migrates remaining uses of `lazy_static`.

Reply:
Done and done my friend!